netmanager/src/views/components/DataDisplay/DeviceView/DeviceDeployStatus.js

```diff
@@ -336,6 +336,12 @@ export default function DeviceDeployStatus({ deviceData, siteOptions }) {
     site_id: "",
   });
 
+  const deviceStatus = !deviceData.status
+    ? deviceData.isActive === true
+      ? "deployed"
+      : "not deployed"
+    : deviceData.status;
+
   useEffect(() => {
```

Comment:
This looks good, thanks @Codebmk . We might need to review this logic further. Shouldn't the first `?` be `&&`?. As follows:

 ```
const deviceStatus = (!deviceData.status
    && deviceData.isActive === true)
      ? "deployed"
      : "not deployed"
    : deviceData.status;
```

Reply:
```
const deviceStatus = 
!deviceData.status ? (deviceData.isActive === true ? "deployed"  : "not deployed") : deviceData.status;
```

The prettier plugin removed the brackets during `git commit` but my logic was that if we don't have a status, we either check the isActive field and return a new status otherwise, otherwise we return the status for a device that has one.

Reply:
When I insert your logic, it is unable to be used because it has a second if statement but with only the else section...like using `if..else..else...` instead of `if..else..if..else..` [Resource](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator#conditional_chains)

When I introduce the if part to your logic, it works:

const deviceStatus = (!deviceData.status
   && deviceData.isActive === true)
     ? "deployed"
     : "not deployed" ? deviceData.status
   : deviceData.status;

Reply:
Alright, thanks @Codebmk